Tests/TestMessagesViewControllerModel.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  TestMessagesViewControllerModel.swift
+//  MessageKit
+//
+//  Created by Dan Leonard on 8/14/17.
+//  Copyright © 2017 MessageKit. All rights reserved.
+//
```

Comment:
@MacMeDan I think we can just paste it manually for now. I'm still using Xcode 8 but we will automate this in Xcode 9 😃 @lzyraining Thank you for that 👍